code/game/objects/machinery/vending/quick_vendor.dm

```diff
@@ -219,6 +219,10 @@ GLOBAL_LIST_INIT(quick_loadouts, init_quick_loadouts())
 			if(user_id.flags_id & CAN_BUY_LOADOUT)
 				user_id.flags_id &= ~CAN_BUY_LOADOUT
 				selected_loadout.quantity --
+				for(var/obj/item/inventory_items in ui.user)
+					if(inventory_items.flags_equip_slot == ITEM_SLOT_ID)
+						continue
+					ui.user.dropItemToGround(inventory_items)
```

Comment:
?

Reply:
Doing this so that the items you have equipped aren't deleted as reported here https://github.com/tgstation/TerraGov-Marine-Corps/issues/14417

Reply:
and why are you doing it to the hvh vendor

Reply:
I don't see any reason not to. Unless you want the HvH vendor to delete pre-existing items?

Reply:
that is why I coded it that way lol